src/main/datalog_console/components/tree_table.cljs

```diff
@@ -1,57 +1,72 @@
 (ns datalog-console.components.tree-table
-  (:require [reagent.core :as r]))
+  (:require [reagent.core :as r]
+            [clojure.set]))
 
 (declare tree-table)
 
-(defn table-row [{:keys [level row expandable-row? expand-row render-col full-width?] :as props}]
+(defn table-row [{:keys [level row expandable-row? expand-row render-col full-width? view-state table-id] :as props}]
```

Comment:
are view-state and table-id still needed in addition to the `^{:key …}` in the parent component? I’d prefer to reduce complexity if they are not needed

Reply:
Did you try moving these props down into the `(fn […` expression? It seems like they should be down there

Reply:
Fixed in refactor during pairing